tests/e2e/testing-manifests/statefulset/nginx/statefulset.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: apps/v1beta2
```

Comment:
the `apps/v1` API has been available since K8s 1.9. `v1beta2` will removed in 1.16 (so, very very soon). Probably better to change this now to avoid future issues.

There's a note about it in the [1.15](https://kubernetes.io/docs/setup/release/notes/#additional-notable-feature-updates) release notes, and a similar note was in there for 1.14.